doc/source/examples/sharp-immersed-boundary/flow-around-particle-sharp/flow-around-particle-sharp.rst

```diff
@@ -0,0 +1,209 @@
+The domain makes use of an inlet velocity of :math:`1~m/s` on the left, slip boundary conditions parallel to the flow direction, and an outlet on the right of the domain. 
+
+.. code-block:: text
+
+    subsection boundary conditions
+        set number = 5
+        subsection bc 0
+            set id   = 0
+                set type = function
+            subsection u
+                set Function expression = 1
+            end
+            subsection v
+                set Function expression = 0
+            end
+            subsection w
+                set Function expression = 0
+            end
+        end
+        subsection bc 1
+            set id   = 2
+            set type = slip
+        end
+        subsection bc 2
+            set id   = 3
+            set type = slip
+        end
+        subsection bc 3
+            set id   = 4
+            set type = slip
+        end
+        subsection bc 4
+            set id   = 5
+            set type = slip
+        end
+    end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~
+
+In order to produce a varying Reynolds number, we vary the kinematic viscosity of the fluid. We define the Reynolds number as :math:`Re = \frac{UD}{\nu}` where :math:`U` and :math:`D` are fixed at 1. Therefore, 
+the kinematic viscosity is defined as :math:`nu = \frac{1}{Re}`. The following block defines the viscosity for a Reynolds number of 1.
+
+.. code-block:: text
+
+    subsection physical properties
+        subsection fluid 0
+            set kinematic viscosity = 1
+        end
+    end
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~
+
+With the Reynolds number ranging from 0.1 to 1, the simulation can easily be solved by using a steady-state solver. The mesh is sufficiently refined to capture the flow around the particle and therefore doesn't 
+require any mesh adaptation.
+
+.. code-block:: text
+
+    subsection simulation control
+        set method            = steady
+        set number mesh adapt = 2
+        set output path       = ./
+        set output name       = out
+        set output control    = iteration
+        set output frequency  = 1
+        set subdivision       = 1
+    end
+
+---------------
+Results
+---------------
+
+The simulation of this case results in the following solution for the velocity and pressure field.
+
+Velocity:
+ 
+.. image:: images/flow-around-particle-sharp-velocity.png
+    :alt: Velocity field around the particle
+    :align: center
+
+Pressure: 
+
+.. image:: images/flow-around-particle-sharp-pressure.png
+    :alt: Pressure field around the particle
+    :align: center
+
+The drag coefficient is calculated for each Reynolds number using the following formula:
+
+.. math::
+
+    C_d = \frac{F_d}{0.5 \rho U^2 D} = \frac{2f_x}{U^2 D}
+
+Where :math:`F_d` is the drag force, :math:`\rho` is the fluid density, :math:`U` is the velocity, and :math:`D` is the diameter of the particle. The drag force is calculated as the force applied 
```

Comment:
What is rho equal to in this problem?

Reply:
Rho is not specifically defined in this case. The only property we define is the kinematic viscosity. I use the output f_x from lethe which I believe corresponds to F_x/rho.